blocks/library/pullquote/index.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { registerBlock, query as hpq } from 'api';
+import Editable from 'components/editable';
+
+const { children, query } = hpq;
+
+registerBlock( 'core/pullquote', {
+	title: wp.i18n.__( 'Pullquote' ),
+	icon: 'format-quote',
+	category: 'common',
+
+	attributes: {
+		value: query( 'blockquote > p', children() ),
+		citation: children( 'footer' ),
+	},
+
+	edit( { attributes, setAttributes, focus, setFocus } ) {
+		const { value, citation } = attributes;
+
+		return (
+			<blockquote className="blocks-pullquote">
+				<Editable
+					value={ value || wp.i18n.__( 'Write Quote…' ) }
+					onChange={
+						( nextValue ) => setAttributes( {
+							value: nextValue
+						} )
+					}
+					focus={ focus && focus.editable === 'value' ? focus : null }
+					onFocus={ () => setFocus( { editable: 'value' } ) }
+				/>
+				{ ( citation || !! focus ) && (
+					<footer>
+						<Editable
+							tagName="footer"
```

Comment:
With `tagName` you no longer need the `<footer>` wrapper. Currently this will render as `<footer><footer></footer></footer>`.